### PR TITLE
Soft page transition animations

### PR DESCRIPTION
**Pull request by pkattera**
# Description

This PR adds page transitions using opacity to soften the appearance as content fades in between pages. This includes the following pages:
- Home page
- Swap page
- Claims page
- Portfolio page
- veBAL page
- Invest flow modal
- Withdraw flow modal
- Migrate flow modal
- Terms of Service
- Privacy Policy
- Cookies Policy

The main change was wrapping content in vue transition tags:
`<Transition appear name="appear"> </Transition>`

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Dependency changes
- [ ] Code refactor / cleanup
- [ ] Documentation or wording changes
- [x] Other: Softer page transition animations

## How should this be tested?

Make sure no pages have broken with the changes. Page content should appear with a soft fade in from opacity: 0 to opacity: 1. 

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have requested at least 1 review (If the PR is significant enough, use best judgement here)
- [x] I have commented my code where relevant, particularly in hard-to-understand areas
- [x] If package-lock.json has changes, it was intentional.
- [x] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

File: src/components/contextual/pages/pool/invest/InvestPage.vue [hunk 1]
```diff
@@ -15,21 +15,23 @@ const { upToLargeBreakpoint } = useBreakpoints();
 </script>
 
 <template>
-  <div class="invest-page-layout-grid">
-    <div v-if="!upToLargeBreakpoint" class="col-span-5">
-      <InvestPageMyWallet />
-    </div>
+  <Transition appear name="appear">
```
Comment (pkattera) [inline src/components/contextual/pages/pool/invest/InvestPage.vue:18]: On pages like this, the only thing that changed was adding the <Transition></Transition> tags.

**Review by garethfuller — CHANGES_REQUESTED** [review]
I believe this should only be wrapped around the components in src/pages and nothing else. I see you've done it around layouts but I don't think that will have any effect when you change pages because the layout mostly stays the same. You have also used in on the InvestPage.vue component which is not an actual page, sorry the naming is misleading.

**Comment by pkattera** [comment]
@garethfuller 
> I believe this should only be wrapped around the components in src/pages and nothing else. I see you've done it around layouts but I don't think that will have any effect when you change pages because the layout mostly stays the same. 

The main reason for it was to apply the same animation for the Navigation. I ended up removing the Transition from defaultLayout and added it to the Navigation component (which lives outside the other pages). 

> You have also used it on the InvestPage.vue component which is not an actual page, sorry the naming is misleading.

I'd like for the first screens of the invest, withdraw and migrate pages/modals to also include the same soft transition. I've applied them to `migrate` and `withdraw` which are actual pages. It looks like the invest page is driven by `InvestPage`. The functionality works as intended using the transition here, but is there a better place for me to add it?

**Comment by garethfuller** [comment]
@pkattera so for example in defaultLayout.vue you would replace `<router-view :key="$route.path" />` with:
```vue
<router-view v-slot="{ Component, route }">
  <transition name="appear">
    <component :is="Component" />
  </transition>
</router-view>
```
And then you can do the same BlankLayout, ContentLayout and FocussedLayout.

**Comment by pkattera** [comment]
Closing this PR in favor of #2571

**Pull request closed without merging**